src/nvim/charset.c

```diff
@@ -84,13 +84,13 @@ int buf_init_chartab(buf_T *buf, int global)
   int c2;
```

Comment:
Good catch! Just changed it!
I don't know what happened with this message, I mustve done something wrong..